pandas/tests/apply/test_frame_apply.py

```diff
@@ -995,7 +995,8 @@ def test_consistency_for_boxed(box, int_frame_const_col):
 
 
 def test_agg_transform(axis, float_frame):
-    other_axis = 1 if axis in {0, "index"} else 0
+    axis = float_frame._get_axis_number(axis)
+    other_axis = 1 if axis == 0 else 0
```

Comment:
This makes it so we aren't testing cases (e.g. passing "index" to apply). I don't think we want to do this in the tests, only in the pandas code itself.

Reply:
Changes reverted to original for `test_agg_function` 
Thank you.

Reply:
Thanks @HardevKhandhar. The same remark applies to your change to the other test here. While the changes you made were certainly of the right form, we just don't want to apply them to the tests. I recommend searching the code in the rest of pandas (outside of tests) and find examples to make this change to.

Reply:
Alright, so, I need to revert back to the changes in all the test functions right?

Reply:
If we're making no changes, then I think we can close this PR. But just to be sure - I think you're going in the right direction here, just need to make the modification to cases that occur outside of `pandas.tests`. Once you find some, make the changes similar to what you did here and submit a new PR.

Reply:
Sure, I am closing the PR.
Thanks a lot @rhshadrach :)